plugins/Name_UpperCase.py

```diff
@@ -22,8 +22,7 @@
 from modules.OsmoseTranslation import T_
 from plugins.Plugin import Plugin
 import regex as re
-from modules.downloader import urlread
-import json
+from plugins.modules.name_suggestion_index import _whitelist_from_nsi
```

Comment:
Remove the "_" on starts, it means private function and it is no more.

Reply:
Done, thanks!